src/Sulu/Bundle/MediaBundle/Controller/MediaController.php

```diff
@@ -211,6 +211,99 @@ public function deleteAction($id)
     }
 
     /**
+     * Creates a new preview image and saves it to the provided media.
+     *
+     * @Post("media/preview/{id}")
```

Comment:
That's not REST conform, should be solved as child resource, as in e.g. https://github.com/sulu-io/sulu/blob/develop/src/Sulu/Bundle/ContactBundle/Controller/ContactMediaController.php